test/js/pageobjects/CrossDeviceUploadsSuccessful.js

```diff
@@ -0,0 +1,22 @@
+import Base from './BasePage.js'
+import {locale, verifyElementCopy} from '../utils/mochaw'
+
+class CrossDeviceUploadsSuccessful extends Base{
+
+  get uploadsSuccessfulTitle() { return this.$('.onfido-sdk-ui-PageTitle-titleSpan')}
+  get uploadsSuccessfulSubtitle() { return this.$('.onfido-sdk-ui-PageTitle-titleWrapper > div:nth-child(2)')}
```

Comment:
Same, change to use the class instead of `nth-child`

Reply:
Done.